product-updates/insights-3-2-release-note.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SysKit Insights 3.2.0 - Release Note
+description: >-
+  This article describes what’s new and improved in the latest version of SysKit
+  Insights.
+author: Hrvoje Bagaric
+date: 10/12/2019
+---
+
+# SysKit Insights 3.2.0
+
+This is a minor release and it fixes critical bugs and brings some quality of life improvements.

```

Comment:
This is a minor release that comes with critical bug fixes and brings some quality of life improvements.